parlai/scripts/train_model.py

```diff
@@ -221,6 +221,15 @@ def setup_args(parser=None) -> ParlaiParser:
         choices=['max', 'min'],
         help='the direction in which to optimize the validation metric, i.e. maximize or minimize',
     )
+    train.add_argument(
+        '-vmf',
+        '--validation-metric-fn',
```

Comment:
could we change the long version to be `--validation-metric-aggregation-function`? as that's more descriptive (I was confused at first)